changes/3551.misc

```diff
@@ -0,0 +1 @@
+Implement Documents, Desktop & Pictures paths in the Dummy backend and update the simple test app to print them.
```

Comment:
A couple of comments here:
* This should be `3551.feature.rst` file - it's a new feature, not a miscellanous update; and it's RST content
* It should be phrased in the form of a release note:
```suggestion
App paths now expose locations for Documents, Desktop & Pictures folders.
```